types/aws-lambda/trigger/api-gateway-authorizer.d.ts

```diff
@@ -36,18 +36,15 @@ export interface APIGatewayTokenAuthorizerEvent {
 // Worse, it will include "body" and "isBase64Encoded" properties, unlike the real call!
 export interface APIGatewayRequestAuthorizerEvent {
     type: "REQUEST";
+    methodArn: string;
     resource: string;
     path: string;
     httpMethod: string;
     headers: { [name: string]: string } | null;
-    multiValueHeaders: { [name: string]: string[] } | null;
     pathParameters: { [name: string]: string } | null;
     queryStringParameters: { [name: string]: string } | null;
-    multiValueQueryStringParameters: { [name: string]: string[] } | null;
-    stageVariables: { [name: string]: string } | null;
     requestContext: APIGatewayEventRequestContextWithAuthorizer<undefined>;
-    domainName: string;
-    apiId: string;
```

Comment:
Just double-checked, these two are on the requestContext, clearly me misreading the nesting, but the multiValue* fields definitely are sent in the real request. Same situation as identity above.